templates/kubernetes/4/rancher-compose.yml

```diff
@@ -22,6 +22,9 @@ etcd:
         increment: 1
         max: 3
         min: 1
+    metadata:
```

Comment:
There are no changes to etcd image, yet this new field is added. Who processes it? Were there any changes to the script?